### DISCUSSION TITLE
use variable in component's props

### DISCUSSION THREAD

**Discussion by sqllyw** [vuejs/petite-vue#136] [General]
I have something like this:

````
 <div v-scope="Counter({ initialCount: start })" @vue:mounted="mounted">
            <p>{{ count }}</p>
            <button @click="inc">increment</button>
        </div>
````

start is a variable in the createApp object, it got initialized in the mounted event, but the component Counter will not pick up the value, it uses the original value, any idea how to make this a little 'reactive'? thanks.

https://jsfiddle.net/c15ojLa2/

**Comment by nfplee** [comment]
Here's an example of how to do this:

https://github.com/vuejs/petite-vue/discussions/47
